api/AbortController.json

```diff
@@ -0,0 +1,208 @@
+{
+  "api": {
+    "AbortController": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AbortController",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
```

Comment:
'false', see https://bugs.chromium.org/p/chromium/issues/detail?id=750599 (will be shipped very soon)